src/app/reporting/Engine.cpp

```diff
@@ -204,16 +204,16 @@ CHIP_ERROR Engine::BuildSingleReportDataAttributeReportIBs(ReportDataMessage::Bu
                                       pathForRetrieval, &encodeState);
             if (err != CHIP_NO_ERROR)
             {
-                ChipLogError(DataManagement,
-                             "Error retrieving data from clusterId: " ChipLogFormatMEI ", err = %" CHIP_ERROR_FORMAT,
-                             ChipLogValueMEI(pathForRetrieval.mClusterId), err.Format());
-
                 // If error is not an "out of writer space" error, rollback and encode status.
                 // Otherwise, if partial data allowed, save the encode state.
                 // Otherwise roll back. If we have already encoded some chunks, we are done; otherwise encode status.
 
                 if (encodeState.AllowPartialData() && IsOutOfWriterSpaceError(err))
                 {
+                    ChipLogError(DataManagement,
+                        "Encoding is out of space, and partial data allowed, save the encode state for report chunking on clusterId: " ChipLogFormatMEI,
```

Comment:
is this an actuall `Error` ? why do we add `save the encode state ...` into the error text?

Reply:
not error, we need mention it is processing some metadata durinng report chunking.